### ISSUE TITLE
Does not record `else` block if not part of a rescue

### ISSUE THREAD

**Issue by JoshCheek** [JoshCheek/seeing_is_believing#73]
This works as expected, it records the `2` in the else block.

``` ruby
def a
rescue
else 123  # => 123
end       # => :a
a         # => 123
```

However, if we remove the `rescue`, it does not record the `else`.

``` ruby
def a
else 123
end       # => :a
a         # => 123
```

Looks like Parser omits it from the AST, I opened an issue [here](https://github.com/whitequark/parser/issues/304#issuecomment-234805261).

My initial diagnosis was this, which may still be relevant at some point in the future: Probably an issue with `SeeingIsBelieving::WrapExpressions`, the test should go [here](https://github.com/JoshCheek/seeing_is_believing/blob/0009cfdf1fe616e2300effab7f108475ee503919/spec/wrap_expressions_spec.rb#L772-L776), or somewhere close by. To fix it, probably follow the recursive invocations through [this](https://github.com/JoshCheek/seeing_is_believing/blob/0009cfdf1fe616e2300effab7f108475ee503919/lib/seeing_is_believing/wrap_expressions.rb#L96) case statement, and figure out where it abandons the else clause.

If this gets fixed, remove [this](https://github.com/JoshCheek/seeing_is_believing/blob/0009cfdf1fe616e2300effab7f108475ee503919/Readme.md#known-issues) comment from the Readme.

**Comment by JoshCheek** [comment]
Fix should have been part of 2.3.1.3 ([here](https://github.com/whitequark/parser/blob/380dca3161c828756dd0447061fb406b0a3fef10/CHANGELOG.md#v2313-2016-09-17)).

But it looks like it's still an issue, I opened a second issue [here](https://github.com/whitequark/parser/issues/320).

**Comment by whitequark** [comment]
This is fixed now.
